tests/apollo/util/bft.py

```diff
@@ -714,19 +734,24 @@ def start_replica_cmd(self, replica_id):
             keys_path = os.path.join(self.txn_signing_keys_base_path, "transaction_signing_keys")
             cmd.append("-t")
             cmd.append(keys_path)
-        return cmd
+        return cmd, cmd[replica_binary_path_index]
 
     def check_error_logs(self):
         """
         Checking FATAL logs in replica logs and writing in a file.
         If the file exists for any testcase, warning is displayed in CI
         """
+
+        if not self.current_test_case_path:
+            # Some tests to not start any replicas/cre, or with KEEP_APOLLO_LOGS=FALSE - in that case there is nothing to check
```

Comment:
I would still write to the eliot log file that this is an empty check (Imagine someone running with KEEP_APOLLO_LOGS=FALSE without knowing)